IO.Eventuate.Tram.Tests/docker-compose.yml

```diff
@@ -1,6 +1,16 @@
 ---
 version: '3.4'
 services:
+  dbsetup:
```

Comment:
PLease look at https://github.com/eventuate-tram/eventuate-tram-core-dotnet/blob/c7259f4e03345492253d5d8fe3af7d40edfe1262/IO.Eventuate.Tram.IntegrationTests/docker-compose.yml#L3

You just need to copy it.

In particular you need the `volumes` to point to the SQL files to initialize the database and you need add those SQL files to the repository

Reply:
Done